src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-sidebar/entity-viewer-sidebar-toolstrip/EntityViewerSidebarToolstrip.tsx

```diff
@@ -90,11 +89,10 @@ export const EntityViewerSidebarToolstrip = () => {
         image={shareIcon}
       />
       <ImageButton
-        status={Status.DISABLED}
-        description="Downloads"
+        status={getViewIconStatus(SidebarModalView.DOWNLOADS)}
+        description={SidebarModalView.DOWNLOADS}
```

Comment:
Not important at the moment, but `description` is free text that doesn't need to be captured in enums. It's just a lucky chance that the descriptions for these buttons currently happen to be the same as our internal strings representing their statuses.

Reply:
That's done now ✅